src/mpid/ch4/netmod/ucx/ucx_pre.h

```diff
@@ -30,7 +30,9 @@ typedef union {
 
 typedef struct {
     int handler_id;
+    ucp_datatype_t datatype;
```

Comment:
@raffenet right now this field is not used anywhere. Should I remove it?

Reply:
yes, get rid of it